src/main/java/com/ericsson/ei/subscriptionhandler/InformSubscription.java

```diff
@@ -176,6 +192,12 @@ private String prepareMissedNotification(String aggregatedObject, String subscri
         return document.toString();
     }
 
+    private String getSubscriptionField(String fieldName, JsonNode subscriptionJson) {
+        String value = subscriptionJson.get(fieldName).toString().replaceAll(REGEX, "");
+        LOGGER.debug(fieldName + " : " + value);
+        return value;
```

Comment:
I think its more clearer in the log if you add some description name before " fieldname : value". In the log it can be hard to see where and what this log message is for.